### ISSUE TITLE
Disable panel's shadow 

### ISSUE THREAD

**Issue by usimarit** [material-shell/material-shell#420]
**Describe the solution you'd like**
It would be nice if you make a button that toggles shadow under both top and bottom panels.

**Comment by PapyElGringo** [comment]
This is a solution but what is the problem?

**Comment by usimarit** [comment]
@PapyElGringo Oh sorry, it's a feature request, not a problem.

**Comment by PapyElGringo** [comment]
Yeah I understood but I think it's to fix an underlying problem you have no? It's just a visual concern?

**Comment by usimarit** [comment]
@PapyElGringo Yeah it's just for the beauty.

**Comment by wjbuys** [comment]
Yes please. This visually obscures the corner of a full screen terminal at a small font size enough to be annoying, I feel like my windows are hidden under a thick bezel or something :(

![image](https://user-images.githubusercontent.com/31602/132083468-dd0b3200-280f-45a0-9638-1040d5996fb0.png)

**Comment by iamarcel** [comment]
Would love this too. I don't like how my window manager is covering every window it's managing; the shell should go out of the way to make place for the content. Also visuospatially, IMO, it would make more sense that the tab panels are at the same depth as the windows it's managing, so it feels like it's "holding" the windows instead of floating above it.

**Comment by agriffis** [comment]
I'd appreciate this too. The shadow makes it feel like the bars are dominating and encroaching on the application.
